src/lib/Libutil/work_task.c

```diff
@@ -103,24 +103,61 @@ struct work_task *set_task(enum work_type type, long event_id, void (*func)(stru
 	pnew->wt_aux2  = 0;
 
 	if (type == WORK_Immed)
-		append_link(&task_list_immed, &pnew->wt_linkall, pnew);
+		append_link(&task_list_immed, &pnew->wt_linkevent, pnew);
 	else if (type == WORK_Timed) {
 		pold = (struct work_task *)GET_NEXT(task_list_timed);
 		while (pold) {
 			if (pold->wt_event > pnew->wt_event)
 				break;
-			pold = (struct work_task *)GET_NEXT(pold->wt_linkall);
+			pold = (struct work_task *)GET_NEXT(pold->wt_linkevent);
 		}
 		if (pold)
-			insert_link(&pold->wt_linkall, &pnew->wt_linkall, pnew,
+			insert_link(&pold->wt_linkevent, &pnew->wt_linkevent, pnew,
 				LINK_INSET_BEFORE);
 		else
-			append_link(&task_list_timed, &pnew->wt_linkall, pnew);
+			append_link(&task_list_timed, &pnew->wt_linkevent, pnew);
 	} else
-		append_link(&task_list_event, &pnew->wt_linkall, pnew);
+		append_link(&task_list_event, &pnew->wt_linkevent, pnew);
 	return (pnew);
 }
 
+/**
+ *
+ * @brief
+ * 	Convert a work task to the type specified.
+ *
+ * @param[in]	ptask	- task being converted
+ * @param[in]	wtype	- work task type to convert
+ *
+ * @return int
+ * @retval 0: success
+ * @retval -1: failure
+ */
+int
+convert_work_task(struct work_task *ptask, enum work_type wtype)
+{
+	pbs_list_head *list;
+
+	if (!ptask)
+		return -1;
+
+	switch (wtype) {
+	case WORK_Immed:
```

Comment:
Is this indent correct?

Reply:
Are you referring to the case falling in the same intent as switch? It is supposed to be like that. There is a snippet in coding standards page: https://openpbs.atlassian.net/wiki/spaces/DG/pages/1481670667/PBS+Pro+Coding+Standards